jumanji/environments/packing/bin_pack/conftest.py

```diff
@@ -148,7 +148,119 @@ def __call__(self, key: chex.PRNGKey) -> State:
         )
 
 
-class DummyConstrainedGenerator(DummyGenerator):
+class DummyExtendedGenerator(DummyGenerator):
+    """Dummy instance generator used for testing. It outputs a constant instance with a 20-ft
+    container and 3 items: two identical items and a different third one to be able to
+    test item aggregation.
```

Comment:
I will update the comment, but when merging the support MR , this comment will stay the same since it is correct in this case, I removed the bit about item aggregation though.